model/src/main/java/org/gluu/oxtrust/model/registration/ManagementRegReqParam.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * oxTrust is available under the MIT License (2008). See http://opensource.org/licenses/MIT for full text.
+ *
+ * Copyright (c) 2014, Gluu
+ */
+package org.gluu.oxtrust.model.registration;
+
+import org.apache.commons.lang.StringUtils;
+
+/**
```

Comment:
I don't think we should add this kind of comments here.
This means we will never use this enum outside of RegistrationManagementService, if this is the case then make it inner class/enum. or remove the comment and the SuppressWarnings please.